app/views/admin/worldwide_organisations_translations/edit.html.erb

```diff
@@ -1,17 +1,44 @@
-<% page_title "Edit translation for: #{@english_worldwide_organisation.name}" %>
-<div class="row">
-  <div class="col-md-8">
-    <h1>Edit ‘<%= @translation_locale.native_language_name %> (<%= @translation_locale.english_language_name %>)’ translation for: <%= @english_worldwide_organisation.name %></h1>
+<% content_for :page_title, "Edit translation" %>
+<% content_for :title, "Edit translation" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @english_worldwide_organisation)) %>
 
-    <%= form_for @translated_worldwide_organisation, url: admin_worldwide_organisation_translation_path(@translated_worldwide_organisation, translation_locale), method: :put, html: {class: 'well'} do |form| %>
-      <%= form.errors %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @translated_worldwide_organisation, url: admin_worldwide_organisation_translation_path(@translated_worldwide_organisation, translation_locale), method: :put do |form| %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Name (required)"
+        },
+        name: "worldwide_organisation[name]",
+        id: "worldwide_organisation_name",
+        value: @translated_worldwide_organisation.name,
+        heading_level: 2,
+        heading_size: "l",
+        error_items: errors_for(form.object.errors, :name),
+        right_to_left: @translated_worldwide_organisation.translation_locale.rtl?,
+        right_to_left_help: false
+      } %>
 
-      <%= form.translated_text_field :name %>
+      <div class="govuk-!-margin-bottom-8">
+        <h2 class="app-view-translation__english-content govuk-heading-m govuk-!-margin-bottom-2">English:</h2>
+        <p class="app-view-translation__english-content govuk-body"><%= @english_worldwide_organisation.name %></p>
+      </div>
 
-      <%= form.save_or_cancel cancel: admin_worldwide_organisation_translations_path(@english_worldwide_organisation) %>
+      <div class="govuk-button-group">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Save",
+          data_attributes: {
+            module: "gem-track-click",
+            "track-category": "form-button",
+            "track-action": "worldwide-organisation-button",
+            "track-label": "Save"
+          }
+        } %>
+
+        <%= link_to "Cancel", admin_worldwide_organisation_translations_path(@english_worldwide_organisation), class: "govuk-link govuk-link--no-visited-state" %>
+      </div>
     <% end %>
   </div>
-  <div class="col-md-4">
-    <%= legacy_simple_formatting_sidebar %>
-  </div>
+
+  <div class="govuk-grid-column-one-third"></div>
```

Comment:
I think this can probs come out since it's not populated with anything right?

Reply:
Yes, that's true, it did include the formatting stuff but we removed it. 👍